### ISSUE TITLE
Add technology news from tweakers.net for Dutch News Channel

### ISSUE THREAD

**Issue by larsenv** [RiiConnect24/File-Maker#7]
It's been requested that we change the technology news source to tweakers.net for the Dutch News Channel.

**Comment by larsenv** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Add technology news from tweakers.net for Dutch News Channel](https://trello.com/c/8cqLDU8z/3-add-technology-news-from-tweakers.net-for-dutch-news-channel)

**Comment by ValerieKun** [comment]
So does the Forecast accept JavaScript?

**Comment by larsenv** [comment]
No, I don't understand what you mean.

**Comment by ghost** [comment]
@larsenv they're asking if Forecast is a webapp.
@ValerieKun No, it's a binary file parser. 

Also, please refrain from discussing Forecast-related issues on News issues. Thanks!

**Comment by InternalLoss** [comment]
Dear potential Hacktoberfest users; thank you for helping us with this! You are helping Nintendo Wii owners by helping restore our services. Please see the CONTRIBUTING file in the folder for the service on specific information. Thanks!

**Comment by larsenv** [comment]
You know, I am no longer interested in using Tweakers.net for Dutch technology news because:

- A lot of the pictures are in PNGs.
- No specific location for articles, so most articles will be boring without the location on them.
- I feel like the articles are too specific. Like for AP English news we get news about general things happening in the tech world (like for example there's an article about Apple doing something with the iPhone X) but Tweakers includes things, such as "iOS 11.1 is out!"
- The articles seem hard to parse.

Basically, I know most RC24 users love technology but adding Tweakers news to the Dutch News Channel would be like if we somehow included news from three sources - Kotaku, Gizmodo and Engadget and put it into one category, there would be too much of it.
